app/models/compute_resources/foreman/model/openstack.rb

```diff
@@ -35,6 +36,22 @@ def tenant=(name)
       attrs[:tenant] = name
     end
 
+    def allow_ext_net?
+      !(attrs[:allow_ext_net] == 0)
+    end
+
+    def allow_ext_net
+      attrs[:allow_ext_net]
+    end
+
+    def allow_ext_net=(enabled)
+      if ['true', true, '1', 1].include?(enabled)
```

Comment:
Yeah, that makes sense

On Thu, Jan 28, 2016 at 11:37 AM, Ohad Levy <notifications@github.com>
wrote:

> In app/models/compute_resources/foreman/model/openstack.rb
> <https://github.com/theforeman/foreman/pull/3007#discussion_r51098830>:
>
> > @@ -35,6 +36,22 @@ def tenant=(name)
> >        attrs[:tenant] = name
> >      end
> >
> > +    def allow_ext_net?
> > +      !(attrs[:allow_ext_net] == 0)
> > +    end
> > +
> > +    def allow_ext_net
> > +      attrs[:allow_ext_net]
> > +    end
> > +
> > +    def allow_ext_net=(enabled)
> > +      if ['true', true, '1', 1].include?(enabled)
>
> hmm.. maybe its only for boolean active record fields that match a
> parameter
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/theforeman/foreman/pull/3007/files#r51098830>.
>